yare-core/src/main/java/com/sabre/oss/yare/core/RulesEngineBuilder.java

```diff
@@ -171,6 +177,7 @@ public RulesEngine build() {
                 .withActionMappings(actionMappings)
                 .withFunctionMappings(functionMappings)
                 .withErrorHandler(errorHandler)
+                .withEngineController(engineController)
```

Comment:
You can add tests for this behavior.

Reply:
I've removed from builder. As I discussed with @MarcinLachowicz we want to avoid issue when a client register listener and pass it to builder.